### DISCUSSION TITLE
Multiple Databases

### DISCUSSION THREAD

**Discussion by MarcelMalik** [prisma/prisma#2176] [Q&A] [answered]
I have different PostgreSQL databases. But with Prisma2, I can apparently only use one database per app.

Is there a way to use two or more databases in one node / next.js app with Prisma2?

**Comment by Sytten** [comment]
There is a plugin for multitenant https://github.com/Errorname/prisma-multi-tenant maintained @Errorname!

**Reply by MarcelMalik** [reply]
Thx.
I am trying to work with it as a workaround.
But in my case I do not want to migrate/manage the DBs schemas and that is a requirement at prisma-multi-tenant.

**Reply by Errorname** [reply]
Hi @MarcelMalik ! Don't hesitate to open an issue on prisma-multi-tenant if you want some help on your use case 🙂

**Comment by ryands17** [comment, marked as answer]
Hey @MarcelMalik currently Prisma doesn't support multiple DB's but you can create a Feature request for that here: https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

For the timebeing, you can use the above way that Sytten has suggested.

**Reply by MarcelMalik** [reply]
Thx.
I create a feature request.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
